uber/templates/jobs/staffers.html

```diff
@@ -53,6 +54,7 @@
 {% for attendee in attendees %}
     <tr>
         <td width="20%" data-order="{{ attendee.full_name }}" data-search="{{ attendee.full_name }}"> <a href="../registration/shifts?id={{ attendee.id }}">{{ attendee.full_name }}</a>        </td>
+        {% if c.HAS_ACCOUNTS_ACCESS %}<td width="10">{% if attendee.admin_account %}Enabled{% else %}<input type="checkbox" name="id{{ forloop.counter }}" value="{{ attendee.id }}" onclick="adminList(this.value)"> Check To Enable{% endif %}</td>{% endif %}
```

Comment:
Like we talked about in Slack, you don't need the `name` property here. Add a class instead, and see comment on line 114.